Resources/config/doctrine/StaticPage.orm.yml

```diff
@@ -0,0 +1,52 @@
+Grossum\StaticPageBundle\Entity\StaticPage:
+    type: entity
```

Comment:
I think, we will do something like this:
https://github.com/sonata-project/SonataMediaBundle/blob/f0e6752e633cff5598ec04925bfa51e1a60fe7f4/DependencyInjection/SonataMediaExtension.php#L200
https://github.com/GrossumUA/SonataEasyExtendsBundle/blob/master/DependencyInjection/Compiler/AddMapperInformationCompilerPass.php#L39
https://github.com/GrossumUA/SonataEasyExtendsBundle/blob/master/Mapper/DoctrineORMMapper.php